frontend/src/components/grouping/FuzzyMatchNotificationGroupingComponent.vue

```diff
@@ -114,6 +118,7 @@ const fieldNameToNamingGroups: Record<string, string[]> = {
 };
 
 const fieldNameToLabel: Record<string, string> = {
+  "businessInformation.notOwnedByPerson": "client",
```

Comment:
Since we use "client name" for the "businessInformation.businessName" (4 lines below), should we use the same here, instead of just "client"?

Reply:
This is just to prevent any undefined from being thrown. This specific value will not be shown.